cmd/kops/create_cluster.go

```diff
@@ -985,6 +1005,43 @@ func RunCreateCluster(f *util.Factory, out io.Writer, c *CreateClusterOptions) e
 		return err
 	}
 
+	if c.DryRun {
+		switch c.Output {
+		case OutputYaml:
+
+			var clusters []*api.Cluster
+			cluster.ObjectMeta.CreationTimestamp = v1.NewTime(time.Now())
```

Comment:
It looks like ` kubectl create secret generic dummy --dry-run -oyaml` does not set CreationTimestamp.

Reply:
I can try w/o, but I do not think `kops create -f mycluster.yaml` works w/o it.